### DISCUSSION TITLE
I think this is the right discussion

### DISCUSSION THREAD

**Discussion by MomsaKoder** [community/community#54420] [Programming Help] [answered]
### Select Topic Area

Show & Tell

### Select Feature Area

- [ ] Issues
- [ ] Projects

### Body

I am trying to copy a portfolio setup so i can make adjustments in order to personalize it.  I am using visual studio 2022.

I figured out how to do 90% of it, but when i go to 'build it', I get this

Port "1234" could not be used
Server running at http://localhost:53390/
Building...
× Build failed.

@parcel/package-manager: Could not find module "@parcel/transformer-sass" satisfying 2.8.2.

  C:\Users\bvern\simplefolio\package.json:23:5
    22 |   "devDependencies": {

    23 |     "@parcel/transformer-sass": "^2.8.3",
       |     ^^^^^^^^^^^^^^^^^^^^^^^^^^ Found this conflicting local requirement.

    24 |     "parcel": "^2.8.2",
    25 |     "prettier": "^2.8.1"

Im new at this, and ive had to figure out alot of the codes.  I just dont know what im missing.  I went to discord with this first and no response. thanks in advance

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by spenserblack** [comment, marked as answer]
FYI this is supposed to be for discussions *about* issues and projects, not raising an issue you have :slightly_smiling_face: 
See https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects

But, to answer your question:

[This repo](https://github.com/MomsaKoder/simple-folio), right?

Yeah, I was able to replicate the error by following [their steps](https://github.com/MomsaKoder/simple-folio#how-to-use-) (running `npm audit fix` and manually installing `@parcel/transformer-sass` separately).

But from a fresh clone, by *not* following those instructions and just doing the following, it worked fine for me locally.
```shell
git clone ...
npm install
npm start  # that's it!
```

Try just that and see if it works.

I'd also recommend raising an issue with the author of those docs letting them know that their instructions break in some environments. To be honest, that's the first time I've encountered a "getting started" project that had instructions for fixing their dependencies like that. Usually it's nothing more than clone, `<package manager> install`, and running the start command.

**Comment by MomsaKoder** [comment]
Thank You.  Pretty sure the neighbors heard the victory yell!
